src/typings/targetDescriptor.types.ts

```diff
@@ -106,6 +106,14 @@ export type SimpleScopeTypeType =
   | "xmlElement"
   | "xmlEndTag"
   | "xmlStartTag"
+  // Latex scope types
+  | "part"
+  | "chapter"
+  | "subsection"
+  | "subSubSection"
+  | "paragraph"
```

Comment:
Was hoping to just take this PR to the finish line myself, but got stuck on this one.  The Cursorless scope type "block" (sequence of non-empty lines) internally uses the identifier `paragraph` as well, so this causes a conflict.  I think it will be surprising if the behaviour of “block” changes in LaTeX, especially since empty lines are actually a valid way to create paragraphs.

I decided to call this one `namedParagraph`